### DISCUSSION TITLE
Authenticate Active User

### DISCUSSION THREAD

**Discussion by warsemann** [filamentphp/filament#6249] [Help]
Hi, can i easy check if user is active when login to filament ?

I use Breezy PLugin and try to extends login class but unsuccessful.....

ps : thanks for this beautifull panel :)

**Comment by danharrin** [comment]
Maybe add a middleware in `config/filament.php` to do this?

**Comment by warsemann** [comment]
Yes it's work, thanks !

**Reply by RoseRiyadh** [reply]
how did you do it? can you share the config code, please?
